app/jobs/send_complaint_job.rb

```diff
@@ -0,0 +1,5 @@
+class SendComplaintJob < ApplicationJob
```

Comment:
We need to be careful with our terminology.
We are sending Cases which can be of type "Complaint"

Reply:
Will make them consistent then